Add 11_distance_nearest_port.jl (#9)
gbrrestoration/canonical-reefs: README.md

```TOML
[bathy]
BATHY_DATA_DIR = "path to bathymetry data"  # location of bathymetry raster datasets

[ports]

> PORTS_DATA_DIR = "path to port data" # location of port shapefile dataset

Reviewer: Could I ask why port data is treated separately from other data?

Reviewer 2: I assumed it would be available for people to download on the store or another location like with the Bathymetric data so we don't have to upload it to the repo. Initially I thought it was a larger data folder, but it's just a shapefile. Would you like me to switch to uploading it?

Reviewer: I would just add it to the repo for now.

Ideally, `data` would be empty with the expectation that a future user would collate all the data in the exact directory structure needed. This is to avoid any copyright/license issues. Thankfully I think all the data we use is released under Creative Commons so it shouldn't be an issue right now.

- 0 : no error (does not indicate polygons that only partially overlapped a given reef!)
- 1 : flags that the reef feature did not overlap any satellite data (value set to 7m)
- 2 : flags that the minimum value was above sea level (no changes/adjustments made)

### Coordinate Reference Systems

`1_create_canonical.jl` uses a Great Barrier Reef Features dataset to create the initial

> canoical dataset. These features have a crs EPSG:4283 (GDA1994). These features are then

Reviewer: ```suggestion
canoical dataset. These features are in CRS EPSG:4283 (GDA1994). These features are then
```
